chef/roles/provisioner-server.rb

```diff
@@ -2,12 +2,7 @@
 name "provisioner-server"
 description "Provisioner Server role - Apt and Networking"
 run_list(
-         "recipe[utils]",
-         "recipe[dhcp]",
-         "recipe[nfs-server]",
-         "recipe[provisioner::dhcp_update]",
-         "recipe[provisioner::update_nodes]",
-         "recipe[provisioner::setup_base_images]"
+         "recipe[provisioner::role_provisioner_server]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.